meson.build

```diff
@@ -41,6 +41,13 @@ else
   endif
 endif
 
+# OpenMP
+omp = get_option('with-omp')
+
+if omp
+  omp_dep = dependency('openmp', language:'cpp')
+endif
+
```

Comment:
I updated meson to build with OpenMP support, add -Dwith-omp=true (default false) in the call to meson.py, the number of threads is still set via environment variable (see #830).

Reply:
Thanks! That should make it much easier to build.